runtime/amd64.S

```diff
@@ -1189,20 +1189,22 @@ ENDFUNCTION(G(caml_reperform))
 
 FUNCTION(G(caml_resume))
 CFI_STARTPROC
-    /* %rax -> fiber, %rbx -> fun, %rdi -> arg */
+    /* %rax -> fiber, %rbx -> fun, %rdi -> arg, %rsi -> last_fiber */
         leaq    -1(%rax), %r10  /* %r10 (new stack) = Ptr_val(%rax) */
-        movq    %rdi, %rax      /* %rax := argument to function in %rbx */
+        movq    %rdi, %rax      /* %rax := argument to function in %rdi */
```

Comment:
I disagree with that comment change. The instruction is loading into `rax` the argument to the function, and the function pointer is in `rbx`. If it's confusing, maybe the comment should only say `argument to function`.

Reply:
In other words, the existing comments reads as "argument to (function in rbx)" and you're changing it to "(argument to function) in rdi", both which are equally confusing with the parentheses left out.